core/lib/rom/relation_registry.rb

```diff
@@ -2,17 +2,9 @@
 
 module ROM
   class RelationRegistry < Registry
-    def initialize(elements = {})
+    def initialize(elements = {}, options = {})
       super
       yield(self, elements) if block_given?
     end
```

Comment:
I'm gonna remove this custom constructor as we can now use `Registry#map` to achieve the same thing (it's needed in `FinalizeRelations`